### ISSUE TITLE
Reserve Parallel Licenses causes error

### ISSUE THREAD

**Issue by MorganRodgers** [OSC/bc_osc_ansys_workbench#17]
We've got some code rot surrounding the parallel license reservation.

```
ERROR:  Your job has not been submitted to Owens for the following reason:

	Malformed compound software request "-l software=ansys+1%ansyspar+-3"
	Each software token must be separated by a % character.
	For example:  "-l software=foo+1%bar+1"
```

**Comment by ericfranz** [comment]
See submit.yml.erb

**Comment by ericfranz** [comment]
What steps did you take to reproduce this defect?

**Comment by MorganRodgers** [comment]
I consistently see the defect when I attempt to launch ANSYS workbench with the following settings:

- Version: 2019R2
- Project: PZS0714
- Number of Hours: 1
- Number of Nodes: 1
- Number of Cores: 1
- Reserve ANSYS Parallel License: checked
- Node type: any
- Email on start: unchecked
- License provider: I can use the OSC license

**Comment by johrstrom** [comment]
I was able to submit with parallel licenses.  First with version 19.1 and then with version 2019R2. 

A difference I see off the top is that I'm not in multiple projects, so I didn't specify my project.

**Comment by johrstrom** [comment]
Actually, I triggered it when I specified the number of cores.  Can you remove that setting and see if that's what gets you too?

**Comment by MorganRodgers** [comment]
Thanks for looking at that Jeff. Removing core count allowed me to submit as well.

**Comment by johrstrom** [comment]
I think it could be [this](https://github.com/OSC/bc_osc_ansys_workbench/blob/master/submit.yml.erb#L12) evaluating to be negative? If I submit with 3 cores, it doesn't work, but if I submit with 4 it does. 

It's unlcear why that has something to do with parallel licenses, I guess it doesn't understand `ansyspar+-3` -3 licenses?  If you look at the malformed request and the example, the error has `+-` and the example has just `+`. 

```bash
(num_nodes * num_cpus_per_node_requested) - num_cpus_for_free
1 * 3 - 4 = -1
vs 
1 * 4 - 4 = 0
```

**Comment by MorganRodgers** [comment]
Fixed in 0cc207e9b41167524fe78f63fee98b2352b82ea3.
